tests/run_sboltestrunner.sh

```diff
@@ -38,5 +38,7 @@ java -jar SBOLTestRunner/target/SBOLTestRunner-0.0.1-SNAPSHOT-withDependencies.j
 
 exitcode=$?
 if [ $exitcode -ne 0 ]; then
```

Comment:
Does SBOLTestRunner exit with an error code if it fails I’m not sure it does.

Reply:
I have tested it and looked at the code, I believe it does.